.github/workflows/automate-team-review-assignment.yml

```
@@ -49,7 +49,12 @@ jobs:
 
     assign-reviewers:
       name: Assign reviewers
-      if: ${{ github.event.pull_request && ( github.event.action == 'opened' || github.event.action == 'ready_for_review' ) }}
+      if: >
```

actionlint (1.7.12)

[error] if: condition "${{ github.event.pull_request && (\n  github.event.action == 'opened' ||\n  github.event.action == 'ready_for_review' ||\n  ( github.event.action == 'labeled' && github.event.label.name == 'coderabbit-approved' )\n) }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)